### ISSUE TITLE
Don't always need to return the response

### ISSUE THREAD

**Issue by TheByteSizeDev** [ethansawicki/daily-journal#4]
If you aren't using the return from a fetch call then you don't need to return it. Not every function needs a return just the ones where the returned value is being used.

This talks a little about it: [https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Return_values](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Return_values)

https://github.com/ethansawicki/daily-journal/blob/bfea0a10310d405c18440a4e4c7b8b108fd97406/scripts/dataAccess.js#L28-L42

**Comment by ethansawicki** [comment]
> If you aren't using the return from a fetch call then you don't need to return it. Not every function needs a return just the ones where the returned value is being used.
> 
> This talks a little about it: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Return_values
> 
> https://github.com/ethansawicki/daily-journal/blob/bfea0a10310d405c18440a4e4c7b8b108fd97406/scripts/dataAccess.js#L28-L42

Yeah i think this was from when i was trying to figure out what was going on but good to know!
